src/citations.jl

```diff
@@ -400,18 +419,21 @@ function expand_citation(link::Markdown.Link, meta, page, doc)
                         capitalize=cit.capitalize,
                         starred=cit.starred
                     )
+                    # TODO: There is already a child here, that should not be included
+                    append!(node.children, link_text.children)
```

Comment:
@mortenpi how to handle this?

Reply:
I think if you call `MarkdownAST.unlink!(text_node)`, it should remove the old node from there?

Reply:
Thanks.